### PR TITLE
Remove participant team from challenge

### PR DESCRIPTION
**Pull request by Ram81**
### Description

This PR adds -
- [x] API to remove the participant team from a challenge

@RishabhJain2018

### CODE REVIEW

**Comment by Ram81** [comment]
@RishabhJain2018 if this API is supposed to be used by us then adding checks for `user` being one of the team members won't allow us to use this API right?

File: apps/challenges/urls.py [hunk 1]
```diff
@@ -205,5 +205,10 @@
         r"^phases/(?P<challenge_phase_pk>[0-9]+)/finish_annotation_file_upload/$",
         views.finish_annotation_file_upload,
         name="finish_annotation_file_upload",
-    )
+    ),
+    url(
+        r"^challenge/(?P<challenge_pk>[0-9]+)/team/(?P<participant_team_pk>[0-9]+)/remove_participant_team/$",
```
Comment (RishabhJain2018) [inline apps/challenges/urls.py:210]: `r"^challenge/(?P<challenge_pk>[0-9]+)/team/(?P<participant_team_pk>[0-9]+)/remove_participant_team/$"` -> `r"^challenges/(?P<challenge_pk>[0-9]+)/team/(?P<participant_team_pk>[0-9]+)/remove_participant_team/$",`

File: apps/challenges/views.py [hunk 2]
```diff
@@ -376,6 +376,46 @@ def add_participant_team_to_challenge(
         return Response(status=status.HTTP_201_CREATED)
 
 
+@api_view(["POST"])
+@throttle_classes([UserRateThrottle])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def remove_participant_team_from_challenge(
```
Comment (RishabhJain2018) [inline apps/challenges/views.py:383]: We should add this in `participants` app.

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]

File: apps/participants/views.py [hunk 3]
```diff
@@ -428,3 +430,53 @@ def get_participant_team_details_for_challenge(request, challenge_pk):
...
+
+@api_view(["POST"])
+@throttle_classes([UserRateThrottle])
+@permission_classes((permissions.IsAuthenticated, HasVerifiedEmail))
+@authentication_classes((ExpiringTokenAuthentication,))
+def remove_participant_team_from_challenge(
+    request, challenge_pk, participant_team_pk
+):
+    """
+    API to remoev the participant team from a challenge
```
Comment (RishabhJain2018) [inline apps/participants/views.py:443]: `remoev` -> `remove`

**Review by RishabhJain2018 — CHANGES_REQUESTED** [review]
Minor Change

**Review by RishabhJain2018 — APPROVED** [review]

**Pull request merged**